### DISCUSSION TITLE
something similar for macOS?

### DISCUSSION THREAD

**Discussion by benpeter** [iXab3r/MicSwitch#47] [General]
is anyone aware of a similar tool for macOS?

**Comment by benpeter** [comment]
to answer my own Q - I found
- an Alfred Workflow: https://github.com/blobaugh/Mic-PTT-Alfred-Workflow
- a free app that also shows status in the menu bar https://apps.apple.com/us/app/mutekey/id1509590766?mt=12

**Comment by iXab3r** [comment]
UI of the second one seems quite fancy :) I'll pin it in README file

**Reply by iXab3r** [reply]
Done: https://github.com/iXab3r/MicSwitch
